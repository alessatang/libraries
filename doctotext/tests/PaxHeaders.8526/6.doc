30 atime=1389151474.466882038
30 ctime=1389151474.466882038
