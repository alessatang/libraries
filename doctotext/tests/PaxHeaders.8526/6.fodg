30 mtime=1384978086.894497489
30 atime=1389151474.491881935
30 ctime=1389151474.491881935
