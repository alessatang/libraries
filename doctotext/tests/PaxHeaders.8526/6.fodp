29 mtime=1384978086.89649749
29 atime=1389151474.51988182
29 ctime=1389151474.51988182
