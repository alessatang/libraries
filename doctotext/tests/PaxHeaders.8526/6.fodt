30 mtime=1384978086.899497491
30 atime=1389151474.520881816
30 ctime=1389151474.520881816
