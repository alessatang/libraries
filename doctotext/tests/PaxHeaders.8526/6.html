30 atime=1389151474.532881767
30 ctime=1389151474.532881767
