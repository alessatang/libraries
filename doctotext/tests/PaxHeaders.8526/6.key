30 mtime=1379552625.859474453
30 atime=1389151474.794880691
30 ctime=1389151474.794880691
