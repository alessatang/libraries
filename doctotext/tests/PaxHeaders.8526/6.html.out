30 mtime=1379553522.620585988
30 atime=1389151474.532881767
30 ctime=1389151474.532881767
